deployments/liqo/templates/liqo-wireguard-gateway-client-template.yaml

```diff
@@ -53,6 +53,7 @@ spec:
                 - --leader-election=true
                 {{- end }}
                 securityContext:
+                  privileged: true
```

Comment:
Are capabilities not enough?

Reply:
Nope, we need it to write in **/proc/sys/net/ipv4/ip_forward**